libobs/obs-encoder.c

```diff
@@ -1091,6 +1091,21 @@ void obs_encoder_set_video(obs_encoder_t *encoder, video_t *video)
 		return;
 	}
 
+	if (obs_encoder_scaling_enabled(encoder) &&
+	    obs_encoder_gpu_scaling_enabled(encoder)) {
+		const struct video_output_info *new =
+			video_output_get_info(video);
+
+		if (encoder->scaled_width != new->width ||
+		    encoder->scaled_height != new->height) {
+			blog(LOG_WARNING,
+			     "encoder '%s': Cannot apply a video_t object with"
+			     "different dimensions to a GPU scaled encoder.",
```

Comment:
This looks like this logged line will be joined without a space. Technically logged lines don't need to be broken up (see style guide, "never break user-visible strings such as printk messages"), but making sure the spacing is readable is also fine.